### DISCUSSION TITLE
"getDefaultConfig" is not present at "@rainbow-me/rainbowkit"

### DISCUSSION THREAD

**Discussion by arujjval** [rainbow-me/rainbowkit#2099] [Q&A] [answered]
I am importing a bunch of functions from "@rainbow-me/rainbowkit", but am not able to get "gerDefaultConfig". Rest all are available. 

![Screenshot from 2024-07-15 12-40-17](https://github.com/user-attachments/assets/8c455bfd-b73e-4fe5-a556-7791be9b5e2b)

![Screenshot from 2024-07-15 12-40-30](https://github.com/user-attachments/assets/0fb0d46f-465f-48ee-9c32-d59716706dbf)

whats the issue here?

**Comment by DanielSinclair** [comment]
@arujjval Can you share a reproducible example with your codebase? Can investigate!

**Comment by arujjval** [comment, marked as answer]
Sorry for disturbing but I was getting this issue when I first installed it. After I reinstalled it, the issue was gone. I forgot to close this.
